docs/conf.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
Here should be license header as well as for the all new fines.

Reply:
Ok, I'll add them.

Although I believe that the current license doesn't require all files to contain the license header. At least I cannot see any mention in the license text which actually enforces that, although IANAL. See also [1].

Airflow, being licensed under Apache 2.0 as well, doesn't have license headers in their docs (despite them being an incubating project of the ASF, which, apparently, enforces the rule for having license headers in every single file [2]).

\[1\]: https://softwareengineering.stackexchange.com/questions/270091/is-a-license-header-in-source-files-required-when-applying-the-apache-software-l
\[2\]: http://www.apache.org/legal/src-headers.html

Reply:
Yes, that rule comes from ASF. I'm fine with it and we already follow it.

Reply:
Yeah, but this project is not under ASF, and I doubt that it ever will. Nevertheless, I've already pushed the headers.